iwyu.cc

```diff
@@ -2084,11 +2084,26 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
           const Type* rhs_deref_type = RemoveReference(rhs_type);
           // For derived-to-base pointer conversion to be allowed, the derived
           // type should be complete.
-          // TODO(bolshakov): what about member pointers?
           if (IsDerivedToBasePtrConvertible(rhs_deref_type, lhs_deref_type)) {
             ReportTypeUse(CurrentLoc(), rhs_type, DerefKind::RemoveRefsAndPtr);
             return true;
           }
```

Comment:
`BTT_IsTriviallyAssignable` case has an `else if` here, that would work here too, right?

Reply:
`else` is senseless when the previous branch ends with `return` statement.

Reply:
Ah. Yes.